pkgs/development/python-modules/mkdocs-plugins/default.nix

```
@@ -0,0 +1,64 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  click,
+  essentials-openapi,
+  flask,
+  hatchling,
+  httpx,
+  jinja2,
+  mkdocs,
+  pytestCheckHook,
+  pythonImportsCheckHook,
+  rich,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "mkdocs-plugins";
```

```suggestion
  pname = "neoteroi-mkdocs";
```
https://pypi.org/project/mkdocs-plugins/ doesn't exist

---

Fixed and renamed the folder as well